x-pack/plugins/cloud_security_posture/public/pages/vulnerabilities/vulnerabilities_finding_flyout/vulnerability_json_tab.tsx

```diff
@@ -21,7 +21,7 @@ export const VulnerabilityJsonTab = ({ vulnerabilityRecord }: VulnerabilityJsonT
       style={{ position: 'absolute', inset: 0, top: offsetTopHeight, bottom: offsetBottomHeight }}
       data-test-subj={JSON_TAB_VULNERABILITY_FLYOUT}
     >
-      <CodeEditor
+      <CodeEditor // TODO theme provided in context by x-pack/plugins/cloud_security_posture/public/plugin.tsx
```

Comment:
it works fine!

Reply:
Thanks for the quick turn around on this! Ill remove the TODOs now!